src/main/resources/templates/swift/xcode/exercise/${packageNameFolder}/SampleText.swift

```diff
@@ -0,0 +1,13 @@
+import Foundation
+
+class SampleText {
+    var text: String
```

Comment:
Shouldn't it be private, since it is stated as private in the Problem Statement?
```suggestion
    private var text: String
```

Reply:
you are right, it is not private because of the Test setup, as I explained below

Reply:
Yes, that's true.